RadixWallet/Clients/OverlayWindowClient/InfoLink.swift

```diff
@@ -0,0 +1,129 @@
+import Foundation
+
+// MARK: - OverlayWindowClient.GlossaryItem
+extension OverlayWindowClient {
+	public enum GlossaryItem: String, Sendable {
```

Comment:
This enum should ideally be defined in Sargon so both platforms are in sync.

Reply:
Yeah

Reply:
Adding that in a separate PR